shoop_tests/core/test_methods.py

```diff
@@ -152,23 +152,6 @@ def test_weight_limits():
     assert any(ve.code == "max_weight" for ve in sm.get_validation_errors(source))
 
 
-# TODO: (TAX) Taxing of shipping methods
-@pytest.mark.xfail # TODO: (TAX) Make this test not fail
```

Comment:
Would it be better to make this test actually not fail instead of removing it wholesale? :(